lib/brick_ftp/api/base.rb

```diff
@@ -108,6 +108,12 @@ def method_missing(method_name, *args)
 
         read_property(method_name)
       end
+
+      def display
```

Comment:
I don't like this but I don't see another way to access it dynamically.  Display is defined on Object so method_missing is never hit.